pankaj/golang/assignment1/commands/commands.go

```diff
@@ -0,0 +1,98 @@
+package commands
+
+import (
+	"fmt"
+	"io/ioutil"
+	"log"
+	"os"
+	"os/user"
+	"strings"
+
+	"github.com/pankaj9310/go-k8s-training/pankaj/golang/assignment1/helpers"
+)
+
+func CommandHandler(command string, history *helpers.History) string {
+	cmd := strings.Fields(command)
+	switch cmd[0] {
+
+	case "cd":
+		return getCd(cmd, command)
+
+	case "ls":
+		return getLs()
+
+	case "pwd":
+		return getPwd()
+
+	case "history":
+		return getHistory(history)
+
+	default:
+		return fmt.Sprintf("Command %q not found", command)
+	}
+
+}
+func GetTerminal() string {
+
+	user, err := user.Current()
+	if err != nil {
+		log.Fatal("Failed to get the current user information.", err)
+
+	}
+	hostName, err := os.Hostname()
+	if err != nil {
+		log.Fatal("Failed to get host name.", err)
+
+	}
+	getWd, err := os.Getwd()
+	if err != nil {
+		log.Fatal("Failed to get the current working directory.", err)
+	}
+	cwd := strings.Split(getWd, "/")
+	terminal := fmt.Sprintf("%s@%s:~/%s ", user.Username, hostName, strings.Join(cwd[3:], "/"))
```

Comment:
`wd` path will not always strat from `~` . It can also start from `/` depending upon the path of shell execution. Handle accordingly.